<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WLSpindleWidget</class>
 <widget class="QDialog" name="WLSpindleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_12">
     <property name="title">
      <string>Output control</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_19">
      <item row="1" column="0">
       <widget class="WLEditIOWidget" name="editOutENBSpindle" native="true"/>
      </item>
      <item row="0" column="0">
       <widget class="WLEditIOWidget" name="editOutRUNSpindle" native="true"/>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_10">
     <property name="title">
      <string>Time (S/sec^2)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_15">
      <item row="0" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>accel</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>decel</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="sbDecSOut">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="maximum">
         <double>99999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="sbAccSOut">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="maximum">
         <double>99999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="3" rowspan="2">
    <widget class="QGroupBox" name="gbSOut">
     <property name="title">
      <string>out</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="4" column="0">
       <widget class="QCheckBox" name="cbFastChangeSOut">
        <property name="text">
         <string>fast change</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="WLEditIOWidget" name="editOutSAOUT" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="WLEditIOWidget" name="editOutSPWM" native="true"/>
      </item>
      <item row="0" column="0">
       <widget class="WLEditIOWidget" name="editOutSOUT" native="true"/>
      </item>
      <item row="3" column="0">
       <widget class="WLEditIOWidget" name="editOutSAxis" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="4" rowspan="2">
    <widget class="QGroupBox" name="gbCorrectSOut">
     <property name="title">
      <string>Calc S Out</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_10">
      <item row="0" column="1">
       <widget class="QTableWidget" name="twCalcSout"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WLEditIOWidget</class>
   <extends>QWidget</extends>
   <header>wleditiowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
